src/components/app/MenuButtons.js

```diff
@@ -534,63 +535,87 @@ class ProfileSharingCompositeButton extends React.PureComponent<
           currentButtonIsSecondaryShareButton: isSecondaryShareButtonVisible,
         })}
       >
-        <ProfileSharingButton
-          buttonClassName="menuButtonsShareButton"
-          shareLabel={shareLabel}
-          symbolicationStatus={symbolicationStatus}
-          okButtonClickEvent={this._attemptToShare}
-          shareNetworkUrlCheckboxChecked={this.state.shareNetworkUrls}
-          shareNetworkUrlCheckboxOnChange={this._onChangeShareNetworkUrls}
-          checkboxDisabled={false}
-        />
-        <UploadingStatus progress={uploadProgress} />
-        <ButtonWithPanel
-          className="menuButtonsPermalinkButton"
-          ref={this._takePermalinkButtonRef}
-          label="Permalink"
-          panel={
-            <ArrowPanel
-              className="menuButtonsPermalinkPanel"
-              onOpen={this._onPermalinkPanelOpen}
-              onClose={this._onPermalinkPanelClose}
-            >
-              <input
-                type="text"
-                className="menuButtonsPermalinkTextField"
-                value={shortUrl}
-                readOnly="readOnly"
-                ref={this._takePermalinkTextFieldRef}
+        {/* display only the button that is visible */}
+        {state === 'local' ? (
+          <CSSTransition classNames="animateDown" timeout={200} in={true}>
```

Comment:
I'm a but puzzled, because I think we don't need to use `in` if we use the `TransitionGroup`. So maybe (maybe ?) the `TransitionGroup` isn't needed, and so all `CSSTransition` with `timeout={0}` could be removed...